test/integration/features/proxy.feature

```diff
@@ -0,0 +1,61 @@
+@proxy @minishift-only
+Feature: Minishift can run behind proxy
+  As a user I can use Minishift behind a proxy.
+  INFO: Code behind this feature will start a proxy server in background
+        on port 8181 and will try to get IP of host machine automatically
+        to set MINISHIFT_HTTP_PROXY for this feature only.
+        However one can override port number or set IP manually if needed.
+        To use custom port number run this test with environmental variable
+        INTEGRATION_PROXY_CUSTOM_PORT.       
+        To use custom IP address please set INTEGRATION_PROXY_CUSTOM_IP
+        environmental variable.
+
+  Scenario: Start behind the proxy
+   Given user starts proxy server and sets MINISHIFT_HTTP_PROXY variable
+    When executing "minishift start" succeeds
+    Then proxy log should contain "Accepting CONNECT to registry-1.docker.io:443"
+     And proxy log should contain "Accepting CONNECT to auth.docker.io:443"
+
+  Scenario: Proxy environmental variable is set in created VM
+    When executing "minishift ssh -- cat /var/lib/boot2docker/profile" succeeds
```

Comment:
I believe this is now ISO specific. I will change this a bit to make it ISO independent.